client/src/main/java/org/evosuite/Properties.java

```diff
@@ -1242,11 +1244,11 @@ public enum TestNamingStrategy {
     public static TestNamingStrategy TEST_NAMING_STRATEGY = TestNamingStrategy.NUMBERED;
 
     public enum VariableNamingStrategy {
-        TYPE_BASED
+        TYPE_BASED, HEURISTICS_BASED
     }
 
     @Parameter(key = "variable_naming_strategy", group = "Output", description = "What strategy to use to derive names for variables")
-    public static VariableNamingStrategy VARIABLE_NAMING_STRATEGY = VariableNamingStrategy.TYPE_BASED;
+    public static VariableNamingStrategy VARIABLE_NAMING_STRATEGY = VariableNamingStrategy.HEURISTICS_BASED;
```

Comment:
I'm inclined to keep the variable naming strategy as `TYPE_BASED`, by default, for now.

Reply:
Yes, definitely, I'm sorry, I thought I uploaded it with `TYPE_BASED` by default, the heuristics proposed are an option. I'll correct it right away.

Reply:
Thanks.